src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -177,25 +182,48 @@ private function isOtherServerProcessRunning($address)
     }
 
     /**
+     * Determine the absolute file path for the router script based on user input and the environment.
+     *
+     * @param InputInterface  $input  An InputInterface instance
+     * @param OutputInterface $output An OutputInterface instance
+     * @param string          $env    The application environment
+     *
+     * @return string|bool The absolute file path of the router script, or false on failure
+     */
+    private function determineRouterScript(InputInterface $input, OutputInterface $output, $env)
+    {
+        $router = $input->getOption('router');
+
+        if (null === $router) {
+            $router = $this
+                ->getContainer()
+                ->get('kernel')
+                ->locateResource(sprintf('@FrameworkBundle/Resources/config/router_%s.php', $env))
+            ;
+        } else {
+            if (!file_exists($router)) {
```

Comment:
OK, will merge to `elseif`.

Yep, I've tried this with a relative path and it seems to work fine.  This was actually the main point of this change, and resembles how file checking is done in `ServerRunCommand.php`.